src/components/scenes/Staking/StakeModifyScene.tsx

```diff
@@ -196,6 +199,55 @@ export const StakeModifyScene = (props: Props) => {
       .catch(error => console.log(error))
   }
 
+  const handleStakingFee = (modification: ChangeQuoteRequest['action']) => () => {
```

Comment:
I would prefer the convention `handlePressStakingFree` (always include the event name in the handler; in this case, "Press")

Reply:
done